cli/args/flags.rs

```diff
@@ -26,6 +26,13 @@ use crate::util::fs::canonicalize_path;
 
 use super::flags_net;
 
+/// Validate that the url have a valid file scheme
+/// This check needs to be made before `to_file_path`
+fn have_valid_file_scheme(url: &Url) -> bool {
```

Comment:
Could we use and maybe improve https://github.com/denoland/deno/blob/91dc6fa5f11f5621c21397c783ee899bfd9657ef/cli/util/path.rs#L73 instead?

Reply:
It's seems to take a ModuleSpecifier , I imagine  that's not always the case for urls we use to_file_path on currently

I'll have to check this though

Reply:
How do I use that function inside runtime carte, do I have to duplicate it as well ?

Reply:
@sigmaSd feel free to move this function to `runtime/` directory and export it from there to reuse in the CLI.

Reply:
done but the ci keeps failing randomly